data-structures.md

```diff
@@ -291,8 +291,14 @@ type Bitfield Bytes
 
 ### SectorSet
 
+A sector set stores a mapping of sector IDs to the respective `commR`s.
+
+```sh
+type SectorSet map{Uint:Bytes}
```

Comment:
Suggestion: The spec uses values of type `SectorID` to index into the miner's sector set. This PR (perhaps by accident) specifies that type to be `Uint`. For consistency, I suggest that you specify the `SectorID` type in the `data-structures.md` document.

Reply:
works for me